cmd/aries-js-worker/README.md

```diff
@@ -27,16 +27,90 @@ Go to [`vue-framework-go`](./vue-aries-framework-go) for a sample agent built wi
 
 View [`index.html`](./index.html) for examples.
 
-**Important:** Make sure the web server adds the appropriate headers when serving the compressed `aries-js-worker.wasm.gz` file:
+### Important - Serving the Assets
```

Comment:
Should the PR title be different - I think this PR isn't parameterizing the wasm filename?

Reply:
done